### DISCUSSION TITLE
How to run the project on ESP8266 tested with Epever 4210AN

### DISCUSSION THREAD

**Discussion by Martyello** [softwarecrash/EPEver2MQTT#59] [Show and tell]
When uploading the BIN file to the ESP8266 whith file esp01, all project pinout works with it, however it is necessary to connect the D1 pin cable from the project diagram to D3 pin for proper operation.

**Comment by all-solutions** [comment]
What do you mean? The ESP-01 has neither a D1 nor a D3.
